recipes/libxshmfence/all/conanfile.py

```diff
@@ -38,31 +40,35 @@ def _settings_build(self):
     def _user_info_build(self):
         return getattr(self, "user_info_build", self.deps_user_info)
 
+    def export_sources(self):
+        export_conandata_patches(self)
+
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-    
+
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+            self.options.rm_safe("fPIC")
+        # for plain C projects only
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
```

Comment:
Do you need to bump the `required_conan_version` to `1.53.0` for using `rm_safe`?